File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--programming-against-events-of-a-namedrange-control_3.cs
        private void namedRange1_Change(Microsoft.Office.Interop.Excel.Range Target)
        {
            this.namedRange2.Value2 = "The Change event occurred.";
            this.namedRange2.Font.Color = System.Drawing.ColorTranslator.ToOle(System.Drawing.Color.Blue);
            this.namedRange2.Font.Italic = false;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--programming-against-events-of-a-namedrange-control_3.vb
    Private Sub NamedRange1_Change(ByVal Target As Microsoft.Office.Interop.Excel.Range) _
        Handles NamedRange1.Change

        With Me.NamedRange2
            .Value2 = "The Change event occurred."
            .Font.Color = System.Drawing.ColorTranslator.ToOle(System.Drawing.Color.Blue)
            .Font.Italic = False
        End With
    End Sub